pkg/msr/msr_linux_test.go

```diff
@@ -107,3 +108,30 @@ func TestParseCPUs(t *testing.T) {
 		})
 	}
 }
+
+// This is a hard one to test. But for many systems, 0x3a is a good bet.
+// but this is of necessity not a complete test! We don't want to set an MSR
+// as part of a test, it might cause real trouble.
+func TestTestAndSet(t *testing.T) {
+	if os.Getuid() != 0 {
+		t.Skipf("Skipping test since we are not root")
+	}
```

Comment:
Skip if not x86 and not Intel CPU?

Reply:
well, someday, it should work everywhere. Everything has an MSR. This test is gonna be hard to do anyway, since our msr code is pretty dumb; we don't match against a cpu id range or class or any such thing. This test will not error.